arkane/reference.py

```diff
@@ -322,6 +324,53 @@ def load(self, paths=''):
 
             self.reference_sets[set_name] = reference_set
 
+    def extract_model_chemistry(self, model_chemistry, sets=None):
+        """
+        Return a list of ErrorCancelingSpecies objects from the reference species in the database that have entries for
+        the requested model chemistry
+
+        Args:
+            model_chemistry (str): String that describes the level of chemistry used to calculate the low level data
```

Comment:
We have to be careful with naming convention for `model_chemistry`.  Maybe convert everything to lowercase?  Also, perhaps have a `model_chemistries` attribute for the ReferenceDatabase class that is a list of all the model chemistries in that reference set, and then assert that the `model_chemistry` argument for `extract_model_chemistry` method is in `model_chemistries`

Reply:
Good point. We will have to standardize this, ideally with what Arkane currently uses/will use in the future.